website/docs/r/spanner_instance_iam.html.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: "google"
+page_title: "Google: google_spanner_instance_iam"
+sidebar_current: "docs-google-spanner-instance-iam"
+description: |-
+ Collection of resources to manage IAM policy for a Spanner instance.
+---
+
+# IAM policy for Spanner Instances
+
+Three different resources help you manage your IAM policy for a Spanner instance. Each of these resources serves a different use case:
+
+* `google_spanner_instance_iam_policy`: Authoritative. Sets the IAM policy for the instance and replaces any existing policy already attached.
```

Comment:
Consider a warning note here - we've gotten a user complaint or two that the docs don't strongly enough warn about the consequences of using `foo_iam_policy`.

Reply:
👍 I'll do this for the database IAM docs I'm writing right now, too.